app/components/iframe_component.rb

```diff
@@ -25,6 +25,7 @@ def src
 
   def call
     tag.iframe(src:, height: height_style, width: width_style, title: iframe_title,
-               frameborder: 0, marginwidth: 0, marginheight: 0, scrolling: 'no', allowfullscreen: true)
+               frameborder: 0, marginwidth: 0, marginheight: 0, scrolling: 'no', allowfullscreen: true,
+               allow: 'clipboard-read')
```

Comment:
don't we want to write to the clipboard?

Reply:
Yes, `clipboard-write` works as well, which I believe you are correct that it is preferred.